release_dashboard/lib/widgets/publish_tag_substeps.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../models/conductor_status.dart';
+import '../state/status_state.dart';
+
+/// Group and display all substeps of the release to channel step.
+///
+/// [nextStep] can be executed to proceed to the next step.
+/// Clicking on the publish releaes button will open a dialogue prompt
+/// to confirm if the release tag is correct.
+class PublishTagSubsteps extends StatefulWidget {
+  const PublishTagSubsteps({
+    Key? key,
+    required this.nextStep,
+  }) : super(key: key);
+
+  final VoidCallback nextStep;
+
+  @override
+  State<PublishTagSubsteps> createState() => PublishTagSubstepsState();
+}
+
+class PublishTagSubstepsState extends State<PublishTagSubsteps> {
+  @override
+  Widget build(BuildContext context) {
+    Map<ConductorStatusEntry, Object>? releaseStatus = context.watch<StatusState>().releaseStatus;
+
+    return Column(
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.symmetric(vertical: 20.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: [
+              Text(
+                  'Release tag ${releaseStatus?[ConductorStatusEntry.releaseVersion]} is ready '
```

Comment:
is null a valid value of `releaseStatus` here?

Reply:
It is the same concept with [this](https://github.com/flutter/cocoon/pull/1487#discussion_r762354554)